### ISSUE TITLE
Notebook prefers unlisted packages

### ISSUE THREAD

**Issue by mjwsteenbergen** [dotnet/interactive#652]
### Describe the bug
1. Create new `.dib` file.
2. Create new code block `#r "nuget:Martijn.Extensions"`
3. Run code block
4. See that it installs package 3.0.0
5. Realize that the [latest listed package is 1.13](https://www.nuget.org/packages/Martijn.Extensions/)
6. Be [confused](https://giphy.com/gifs/idk-preity-zinta-confuse-8mndEBLsg9Whg2Sduv)

### Please complete the following:

**Which version of .NET Interactive are you using?** There are a few ways to find this out:

* In a notebook, run the `#!about` magic command. 
* At the command line, run `dotnet interactive --version`.
* In VS Code, run "Report installed version for .NET Interactive` and copy the version number from the status popup.

- OS
    - [x] Windows 10
    - [ ] macOS
    - [ ] Linux (Please specify distro)
    - [ ] iOS
    - [ ] Android
 - Browser 
    - [ ] Chrome
    - [ ] Edge
    - [ ] Firefox
    - [ ] Safari
 - Frontend
    - [ ] Jupyter Notebook
    - [ ] Jupyter Lab
    - [ ] nteract
    - [x] Visual Studio Code
    - [ ] Other (please specify)

**Comment by colombod** [comment]
is the 3.0.0 package present in nuget.org but unlisted?

**Comment by mjwsteenbergen** [comment]
Yes. It is the only unlisted package. Should this be expected behavior?

**Comment by colombod** [comment]
I have just created a console program and then
```
 dotnet add Martijn.Extensions
```

The result is 
![image](https://user-images.githubusercontent.com/375556/88112489-ddd62f80-cba7-11ea-8714-f5de7972a934.png)

While seems to be installing the expected version with visual studio

![image](https://user-images.githubusercontent.com/375556/88112710-550bc380-cba8-11ea-8150-d724d4d5159a.png)

**Comment by jonsequitur** [comment]
This is a NuGet bug: https://github.com/NuGet/Home/issues/7466
